src/config.h

```diff
@@ -149,3 +149,11 @@
 #define MG_EPOLL_ADD(c)
 #define MG_EPOLL_MOD(c, wr)
 #endif
+
+#ifndef MG_ENABLE_STM32H5
+#define MG_ENABLE_STM32H5 0
+#endif
+
+#ifndef MG_ENABLE_STM32H7
+#define MG_ENABLE_STM32H7 0
```

Comment:
These names are a bit confusing to me.
We should add context, OTA perhaps ? I see "sys" below, I guess we should find a name for all this, it is "arch" but "hardware arch"...

Reply:
Those should go, and those are already gone!